cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -965,6 +965,12 @@ def ceph_storage():
 
     ceph_admin = endpoint_from_flag('ceph-storage.available')
 
+    # This is needed as we must allow privileged.
+    reconfigure_apiserver(
```

Comment:
Is there a reason that we are calling this directly instead of using a flag to trigger it?